ChangeLog.d/tls-hs-defrag-in.txt

```diff
@@ -0,0 +1,2 @@
+Changes
+   * Defragment incoming TLS handshake messages.
```

Comment:
I think we can eve make classify that as a bugfix: (1) there is nothing in the spec that says support for receiving fragemented messages is optional (quite the opposite: it's the first point in the [implementation pitfalls](https://datatracker.ietf.org/doc/html/rfc5246#appendix-D.4) section), and (2) it was causing interop failures in practice.

Also, I think we should expand a bit so that users who are not aware of TLS internals may make more sense of the entry, perhaps:
```
Bugfix
   * Support re-assembly of fragmented handshake messages in TLS, as mandated
     by the spec. Lack of support was causing handshake failures with some
     servers, especially with TLS 1.3 in practice (though both protocol
     version could be affected in principle, and both are fixed now).
```

Reply:
done